jetbrains-core/resources/META-INF/ext-python.xml

```diff
@@ -8,4 +8,8 @@
         <handlerResolver runtimeGroup="PYTHON" implementation="software.aws.toolkits.jetbrains.services.lambda.python.PythonLambdaHandlerResolver"/>
         <sam.debugSupport runtimeGroup="PYTHON" implementation="software.aws.toolkits.jetbrains.services.lambda.python.PythonSamDebugSupport"/>
     </extensions>
+
+    <extensions defaultExtensionNs="com.intellij">
+        <directoryProjectGenerator implementation="software.aws.toolkits.jetbrains.ui.wizard.SamInitProjectBuilderPyCharm" />
```

Comment:
We have `python` and `java` packages under the `lambda` namespace for the language specific implementations.

Reply:
Will change to follow the same convention for the `wizard` namespace